libs/containers/AuditLogEntry.lua

```diff
@@ -181,7 +174,7 @@ end
 
 --[=[
 @m getMember
-@r Member
+@r Member/nil
```

Comment:
What is this for?   This is *the* only return in Discordia that is documented with an explicit `nil`.